data/types/base.go

```diff
@@ -30,8 +31,8 @@ const (
 	TagLengthMaximum        = 100
 	TagsLengthMaximum       = 100
 	TimeFormat              = time.RFC3339
-	TimezoneOffsetMaximum   = 7 * 24 * 60  // TODO: Fix! Limit to reasonable values
-	TimezoneOffsetMinimum   = -7 * 24 * 60 // TODO: Fix! Limit to reasonable values
+	TimeZoneOffsetMaximum   = 7 * 24 * 60  // TODO: Fix! Limit to reasonable values
```

Comment:
Is it worth setting to a reasonable value now? Or are we still trying to decide what that should be ....

Reply:
The reasonable values are -12*60 - 14*60. We are working on getting it fixed in the Uploader (work-in-progress), the database, the backend, and the data donation program. It has to be a step-by-step process. Uploader is first, then I'll update the backend and migrate the data. So, this will be fixed some time in the relatively near future.